cmd/helm/installer/install.go

```diff
@@ -113,40 +114,65 @@ func createDeployment(client extensionsclient.DeploymentsGetter, opts *Options)
 
 }
 
-// deployment gets the deployment object that installs Tiller.
-func deployment(opts *Options) (*v1beta1.Deployment, error) {
-	return generateDeployment(opts)
+// Deployment gets the deployment object that installs Tiller.
```

Comment:
Based on our other comment thread, would it be fair to update this docstring to mention that this is only for string representations and should not be submitted to the API?

Reply:
yes, definitely. I'll commit shortly